plugins/@grouparoo/sentry/src/initializers/sentry.ts

```diff
@@ -109,3 +113,15 @@ export class SentryInitializer extends Initializer {
     } catch (e) {}
   }
 }
+
+function cleanupSentryTransaction(
+  transaction: ReturnType<typeof Sentry.startTransaction>
+) {
+  for (const span of transaction["spanRecorder"].spans) {
+    // trim the description (where the SQL transaction would go) down to sentrySpanLengthLimit chars max
+    if (span.description && span.description.length > sentrySpanLengthLimit) {
+      span.description =
+        span.description.slice(0, sentrySpanLengthLimit) + "...";
+    }
+  }
+}
```

Comment:
Now that we have these giant bulk update queries, sentry was dropping larger event payloads.  This truncates our SQL statement lengths so we get *something* at least